fluid/template_panel.fl

```diff
@@ -29,6 +29,9 @@ decl {\#include <stdlib.h>} {private local
 decl {\#include "../src/flstring.h"} {private local
 }
 
+decl {\#include <FL/fl_string.h>} {private local
+}
+
```

Comment:
Builds don't do that in `src` and `fluid` folders because fluid doesn't exist yet when these dirs get compiled in the build process. Hint:
```
for f in *.fl; do
  echo $f
  ../fluid/fluid -u -c $f
done
```

Execute this in `src` and/or `fluid` folders after the build, take care to use the latest fluid w/trailing whitespace fix (`../fluid/fluid`), rebuild everything after that because fluid was changed.

Note: `-u` is optional: it **u**pdates the `.fl` file with the current version etc.

Reply:
Running `fluid -u -c` to recreate the `.fl` and related `{.cxx | .h}` files still needs to be done.
I could also do it afterwards, but maybe you want to give it a try?

Reply:
OK, just did this test. It moved the appearance of the #include for fl_string.h a little higher in the .h file, but otherwise .cxx and .h were unchanged. I'll check it in though, so it doesn't trigger an unexpected diff. ( 3d18d325f )